app/Http/Controllers/Service/AdminController.php

```diff
@@ -0,0 +1,30 @@
+<?php
```

Comment:
Once we know what else the admin will need to do I might get rid of this controller.